### ISSUE TITLE
Help to add a feature in dash-manufacture-spc-dashboard

### ISSUE THREAD

**Issue by mboukabous** [plotly/dash-sample-apps#763]
First of all thank you very much for these really interesting codes and for sharing them.
I'm starting to implement "dash-manufacture-spc-dashboard", my problem is that my data is getting from Python Serial at every interval and I want to be able to add rows on my dataframe at that specific interval, but I don't know where to add the append lines on the code to be able to refresh the dataframe and have more data to display on my charts.
Basically I need to be able to add the data directly on the code and not in the csv file.
Thanks in advance for all the help.
